src/pyflex/config.py

```diff
@@ -215,6 +236,19 @@ def __init__(self, min_period, max_period, stalta_waterlevel=0.07,
             non-overlapping windows. Merging will simply merge overlapping
             windows.
         :type resolution_strategy: str
+
+        :param selection_mode: Strategy to select different types of phases.
+            Possibilities are ``"all_wave"``, ``"body_and_surface_wave"``,
+            ``"body_wave"``, ``"surface_wave"``, ``"mantle_wave"``. If
+            ``"all_wave"``, then pyflex will accept all windows in the
+            signal region(after noise_end). If ``"body_wave"``, the pyflex
+            will only accept windows in the body wave region(after first
+            arrival and before surface wave arrival). If ``"surface_wave"``,
+            pyflex will only accept windows in surface wave region(velocity
+            between min_surface_wave_velocity and max_surface_wave_velocity).
+            If ``"body_and_surface_wave"``, then pyflex will only accept
+            windows inside body and surface wave regions. If ``mantle_wave``,
+            pyflex will only accept windows after the surface wave region.
```

Comment:
Lacks the `:type selection_mode: str` line.

Can you also please change the names to plural, e.g. to `body_waves`, `all_waves`, ... I think that is a bit nicer.

Really minor and its up to you if you want to do it. I will not insist on it: It might be nicer to document it an itemized list:

```
* ``"all_waves"``: All waves in the signal region.
* ``"body_waves"``: ...
```